cmd/cyclonus/README.md

```diff
@@ -1,51 +1,16 @@
-# Cyclonus
+# NetworkPolicy Assistant (derived from Cyclonus)
```

Comment:
Policy Assistant is much better! Changed the dir to `cmd/policy-assistant`.

Still need to adjust Makefile and rename `cmd/policy-assistant/cmd/cyclonus/`. Seems there are 37 non-go files that reference "cyclonus", including some artifacts and CI.

Since the PR is already so large, I'd recommend making those changes in another PR to limit the scope of this one

Reply:
Yep follow up!

Reply:
Tracking here: #171